.circleci/config.yml

```diff
@@ -2562,6 +2639,7 @@ workflows:
           context:
             - circleci-repo-readonly-authenticated-github-token
       - contracts-bedrock-build: # needed for sysgo tests
+          build_args: --skip test
```

Comment:
Just double-checking this is the right flag.  From [the forge docs](https://getfoundry.sh/forge/reference/build/#forge-build) it looks like this will also skip building scripts, which I think we need in some of the test setup:
```
--skip <SKIP>...
          Skip building files whose names contain the given filter.
          
          `test` and `script` are aliases for `.t.sol` and `.s.sol`.
```

Reply:
Discussed offline - looks like this means `script` will disable `.s.sol` files, and `test` will disable `.t.sol` files